Source/WTF/Scripts/Preferences/UnifiedWebPreferences.yaml

```diff
@@ -7919,6 +7919,19 @@ VisualViewportEnabled:
     WebCore:
       default: true
 
+VisuallyContiguousBidiTextSelectionEnabled:
```

Comment:
Can we add conditional: PLATFORM(ISO_FAMILY) given it looks like this option only works in IOS_FAMILY?

Reply:
Sounds good, will add `conditional: PLATFORM(IOS_FAMILY)`.